### DISCUSSION TITLE
Add Pill Icon

### DISCUSSION THREAD

**Discussion by luccawilli** [tailwindlabs/heroicons#892] [Ideas]
Dear HeroIcons Team
I really love your collection and i would appricate if you could add a pill icon.
I know it is always hard to create new icons, so to make it a little easier for you, i have already started myself and attached it here.
![Pill](https://user-images.githubusercontent.com/37955556/206872780-0030c0f5-a779-4e5e-9bcb-e8133bfa35a5.svg)

Maybe you can use it as a template or as inspiration.
Thanks you, 
Best regrads

**Comment by kbrock** [comment]
Suggestion: for things like this, could you put it as a PR?
You can discuss over there.

And if they like the idea, they can then merge.

The way you set this up, if they want to accept this icon, they will have to do work.
If you created a PR, if they want to accept the icon, they just click merge.

BTW/ this is a great icon. I was hoping it was already in the library. Good stuff
Also, I'm not part of the team, so please take my advice with a grain of salt.
